src/html_builder.ts

```diff
@@ -15,486 +15,499 @@ import {
-            HTMLBuilder.get_md_button("point", "Point", POINT_SVG, curmd, ulabel.subtasks),
-            HTMLBuilder.get_md_button("polygon", "Polygon", POLYGON_SVG, curmd, ulabel.subtasks),
-            HTMLBuilder.get_md_button("tbar", "T-Bar", TBAR_SVG, curmd, ulabel.subtasks),
-            HTMLBuilder.get_md_button("polyline", "Polyline", POLYLINE_SVG, curmd, ulabel.subtasks),
-            HTMLBuilder.get_md_button("contour", "Contour", CONTOUR_SVG, curmd, ulabel.subtasks),
-            HTMLBuilder.get_md_button("bbox3", "Bounding Cube", BBOX3_SVG, curmd, ulabel.subtasks),
-            HTMLBuilder.get_md_button("whole-image", "Whole Frame", WHOLE_IMAGE_SVG, curmd, ulabel.subtasks),
-            HTMLBuilder.get_md_button("global", "Global", GLOBAL_SVG, curmd, ulabel.subtasks),
-        ];
-
-        // Append but don't wait
-        $("#" + sp_id + " .toolbox_inner_cls .mode-selection").append(md_buttons.join("<!-- -->"));
-
-        // Show current mode label
-        ulabel.show_annotation_mode(null);
-
-        // Make sure that entire toolbox is shown
-        if ($("#" + ulabel.config["toolbox_id"] + " .toolbox_inner_cls").height() > $("#" + ulabel.config["container_id"]).height()) {
-            $("#" + ulabel.config["toolbox_id"]).css("overflow-y", "scroll");
-        }
+    ulabel.toolbox = toolbox;
 
-        ulabel.toolbox = toolbox;
-
-        // Check an array to see if it contains a ZoomPanToolboxItem
-        let contains_zoom_pan: Function = function(array: unknown[]) {
-            
-            //check if the array is empty
-            if (array.length == 0) return false;
-            
-            // Loop through everything in the array and check if its the ZoomPanToolboxItem
-            for (let idx in array) {
-                if (array[idx] instanceof ZoomPanToolboxItem) {
-                    return true;
-                }
+    // Check an array to see if it contains a ZoomPanToolboxItem
+    let contains_zoom_pan: Function = function(array: unknown[]) {
+        
+        //check if the array is empty
+        if (array.length == 0) return false;
+        
+        // Loop through everything in the array and check if its the ZoomPanToolboxItem
+        for (let idx in array) {
+            if (array[idx] instanceof ZoomPanToolboxItem) {
+                return true;
             }
-            
-            // If the ZoomPanToolboxItem wasn't found then return false
-            return false;
         }
+        
+        // If the ZoomPanToolboxItem wasn't found then return false
+        return false;
+    }
 
-        // Check if initial_crop exists and has the appropriate properties
-        let check_initial_crop: Function = function(initial_crop) {
-
-            // If initial_crop doesn't exist, return false
-            if (initial_crop == null) return false;
+    // Check if initial_crop exists and has the appropriate properties
+    let check_initial_crop: Function = function(initial_crop) {
```

Comment:
A name like `initial_crop_is_valid` or smth would read slightly better, for example
```js
if (initial_crop_is_valid(ulabel.config.initial_crop))
```
i get that this is just a refactor of the og function so just an idea to consider

Reply:
[82ac024](https://github.com/SenteraLLC/ulabel/pull/127/commits/82ac0241d5200a8447103f516dabd17c573c5479)